static/app/views/profiling/profileSummary/index.tsx

```diff
@@ -590,6 +605,13 @@ function AggregateFlamegraphToolbar(props: AggregateFlamegraphToolbarProps) {
         size="xs"
         options={frameSelectOptions}
       />
+      <Button
+        size="xs"
+        onClick={props.onHideRegressionsClick}
+        title={t('Expand or collapse the flamegraph')}
```

Comment:
The title here seems inaccurate as it's not collapsing the flamegraph right? It really toggling the summary information

Reply:
Yes, but the option itself lives in the aggregate flamegraph toolbar, so it's weird to say "expand or collapse y" when y is disjointed and lives someplace else. I also wasn't sure how to call that in a way that would make it obvious as to what it's referencing, so I went with the reverse and added it onto the flamegraph.

Reply:
I'm open to suggestions, but UI wise it's easiest to put this in the toolbar as it's always present. If we were to put it inside the small header on top of the summary it first gets really weird with the content flow and second, we still need to display it somehow in it's collapsed state so that it can be opened again

Reply:
Slept on this and I think we can just say `Expand or collapse the view` here